### ISSUE TITLE
[Webpack] Error: Can't resolve 'babel'

### ISSUE THREAD

**Issue by khovansky-al** [zeit/hyper#998]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS Sierra 10.12.1
- **Hyper.app version**: master branch clone (Latest commit 9c90e19)
- **The issue is reproducible in vanilla Hyper.app**: Is Vanilla

## Issue
After forking and cloning can't run the app in development mode because of the following webpack error (enabled --display-error-details flag to make it more verbose):
```
ERROR in Entry module not found: Error: Can't resolve 'babel' in '/Users/alex/Code/hyper'
resolve 'babel' in '/Users/alex/Code/hyper'
  Parsed request is a module
  using description file: /Users/alex/Code/hyper/package.json (relative path: .)
  after using description file: /Users/alex/Code/hyper/package.json (relative path: .)
    resolve as module
      /Users/alex/Code/node_modules doesn't exist or is not a directory
      /Users/alex/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /Users/alex/Code/hyper/node_modules
        using description file: /Users/alex/Code/hyper/package.json (relative path: ./node_modules)
        after using description file: /Users/alex/Code/hyper/package.json (relative path: ./node_modules)
          using description file: /Users/alex/Code/hyper/package.json (relative path: ./node_modules/babel)
            as directory
              /Users/alex/Code/hyper/node_modules/babel doesn't exist
            no extension
              /Users/alex/Code/hyper/node_modules/babel doesn't exist
            .js
              /Users/alex/Code/hyper/node_modules/babel.js doesn't exist
            .json
              /Users/alex/Code/hyper/node_modules/babel.json doesn't exist
```
Although, if I go to `webpack.config.js` and change
```
module: {
    loaders: [
      {
        test: /\.(js|jsx)$/,
        exclude: /node_modules/,
        loader: 'babel'
      },
```
to
```
module: {
    loaders: [
      {
        test: /\.(js|jsx)$/,
        exclude: /node_modules/,
        loader: 'babel-loader'
      },
```
it works just fine. Any thoughts on why shorthand might be failing?

**Comment by saitonakamura** [comment]
Same story here on Windows

**Comment by matheuss** [comment]
Fixed via #1000 😄 Thanks for reporting!

**Comment by Sherryer** [comment]
but i have set “babel” here in last project，it run well，i wanna know the reasen。
